src/core/metrics.cc

```diff
@@ -156,205 +156,215 @@ Metrics::EnableGPUMetrics()
   }
 
   if (std::getenv("TRITON_SERVER_CPU_ONLY") == nullptr) {
-    singleton->InitializeNvmlMetrics();
+    singleton->InitializeDcgmMetrics();
   }
 
   singleton->gpu_metrics_enabled_ = true;
 }
 
 bool
-Metrics::InitializeNvmlMetrics()
+Metrics::InitializeDcgmMetrics()
 {
 #ifndef TRITON_ENABLE_METRICS_GPU
   return false;
 #else
-  nvmlReturn_t nvmlerr = nvmlInit();
-  if (nvmlerr != NVML_SUCCESS) {
-    LOG_WARNING << "failed to initialize, GPU metrics will not be available: "
-                << nvmlErrorString(nvmlerr);
+  dcgmReturn_t dcgmerr = dcgmInit();
+  if (dcgmerr != DCGM_ST_OK) {
+    LOG_WARNING
+        << "failed to initialize DCGM, GPU metrics will not be available: "
+        << errorString(dcgmerr);
```

Comment:
Is this a dcgm-specific error string method?

Reply:
Yes that is correct